ql/src/java/org/apache/hadoop/hive/ql/lockmgr/DummyTxnManager.java

```diff
@@ -73,11 +76,11 @@ public long openTxn(Context ctx, String user) throws LockException {
 
   @Override
   public boolean isTxnOpen() {
-    return false;
+    return numTxn != 0;
```

Comment:
Why do we need these changes? DummyTxnManager does not support ACID

Reply:
Makes sense. Reverting. Picked up this from Daniel's patch thinking this is used in some cases. Will revisit this change, if some tests fail because of this. But logically speaking, this should not be required, as it doesn't support transactions.

Reply:
We did not reverted this. Why?